day6/salestax.py

```diff
@@ -1,33 +1,68 @@
-import csv
-import pandas
 import fs 
+import csv
+import json
+
+import pandas as pd
+
+class calculation_of_tax:
+
+    def read_File(self):
+        with open('day6/input1.csv', 'r') as file:
+            reader = csv.reader(file, delimiter = '\t')
+            for row in reader:
+                print(row)
+
+    def csv_to_json(self, csvFilePath, jsonFilePath):
+        jsonArray = []
+      
+        #read csv file
+        with open(csvFilePath, encoding='utf-8') as csvf: 
+            #load csv file data using csv library's dictionary reader
+            csvReader = csv.DictReader(csvf) 
+
+            #convert each csv row into python dict
+            for row in csvReader: 
+                #add this python dict to json array
+                jsonArray.append(row)
+  
+        #convert python jsonArray to JSON String and write to file
+        with open(jsonFilePath, 'w', encoding='utf-8') as jsonf: 
+            jsonString = json.dumps(jsonArray, indent=4)
+            jsonf.write(jsonString)
+          
+    def calculate_final_price(self):
+        with open('day6/input2.json') as json_file:
+            data = json.load(json_file)
+
+        indiatax = data[0]["tax"]
```

Comment:
here you have assumed that the first tax is going to be for India but what if you don't know which tax is in the first place. Don't hardcode this part your code should run even if someone changes the input.